### DISCUSSION TITLE
Not an issue, a note on CTkButton

### DISCUSSION THREAD

**Discussion by gowke** [TomSchimansky/CustomTkinter#2851] [Q&A]
The CTkButton does not pass 'event', easily modified in the CTkButton widget in the module, However when you want to capture id(event.widget) in your function you will come across an issue. The TCkButton consists of three elemnts- canvass, image label and text label. Clicking on either will return event for a respective widget, 

Background where this can come up. When you create multiple widgets and a button to manipulate them, but one function which captures the event and identifies which widget is associated with it.  Workaround is to associate three possible event.widget (CTk) on a click instead of one (standard Tk/TTk). The Tk id(event.widget) is replaced by three from CTk.

**Comment by TomSchimansky** [comment]
I don't no if I can do anything about this?

**Comment by gowke** [comment]
Maybe for the future! Great work! Thank you very much for this!
